src/cms/templates/push_notifications/push_notification_list.html

```diff
@@ -8,16 +8,20 @@
     <div class="flex flex-wrap">
         <h1 class="heading">{% trans 'News' %}</h1>
     </div>
-    <div class="flex flex-wrap">
-        <form class="table-search relative w-1/2">
-            <i data-feather="search" class="absolute"></i>
-            <input type="search" placeholder="{% trans 'Search' %}" class="h-full pl-10 pr-4 rounded shadow">
-        </form>
-        <div class="w-1/2 flex flex-wrap justify-end">
+    <div class="flex flex-wrap justify-between">
+        <div class="flex flex-wrap">
+            {% include "generic_language_switcher.html" with target='push_notifications' %}
+            <form class="table-search relative w-1/2">
+                <i data-feather="search" class="absolute"></i>
+                <input type="search" placeholder="{% trans 'Search' %}" class="h-full pl-10 pr-4 rounded shadow">
+            </form>
+        </div>
+        <div class="flex flex-wrap">
             <a href="{% url 'new_push_notification' region_slug=region.slug language_slug=language.slug %}" class="bg-blue-500 hover:bg-blue-600 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline">
-	            {% trans 'Create news message' %}
+                {% trans 'Create news message' %}
             </a>
         </div>
+        </div>
```

Comment:
```suggestion
```
This extra `</div>` breaks the layout of the list:
Before             |  With extra `</div>`
:-------------------------:|:-------------------------:
![Screenshot_2021-05-17 Integreat Redaktionssystem(3)](https://user-images.githubusercontent.com/16021269/118517376-6fb2b880-b737-11eb-815b-080e05278066.png) | ![Screenshot_2021-05-17 Integreat Redaktionssystem(2)](https://user-images.githubusercontent.com/16021269/118517391-72151280-b737-11eb-9aee-6d98bcf12958.png)